app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -228,6 +228,13 @@
   </section>
 <% end %>
 
+<%# Alert %>
+<% if @show_closed_post_office_banner %>
```

Comment:
👏🏻

Reply:
I checked this in user mailer preview with `in_person_proofing_post_office_closed_alert_enabled` set to both true and false.  Both looked good.